fhir-persistence-cos/src/main/java/com/ibm/fhir/persistence/cos/client/COSPropertyConstants.java

```diff
@@ -36,4 +36,7 @@
     
     // The max keys to return per list objects request
     public static final String COS_MAX_KEYS = "max.keys";
+
+    // Compress resources when reading/writing to COS
+    public static final String COS_COMPRESS = "compress";
```

Comment:
we need to make sure this makes its way into the user's guide / wherever else this property group is documented